src/aatoolbox/config/country.py

```diff
@@ -0,0 +1,39 @@
+"""Country configuration setting base class."""
+from pathlib import Path
+
+from aatoolbox.utils.io import parse_yaml
+
+
+class _CodAdmin(object):
+    def __init__(self, base_layer: str = None, **kwargs):
+        self.base_layer = base_layer
+
+
+class _CodABConfig(object):
+    def __init__(self, base_dir: str = None, base_zip: str = None, **kwargs):
```

Comment:
Would it be useful here to add a comment that the `base_dir` and `base_zip` parameter input to `__init__` are automatically extracted from `object`? Or is that basic programming knowledge that I just didn't have?